src/js/utils/browser.js

```diff
@@ -55,11 +100,49 @@ export const ANDROID_VERSION = (function() {
   return null;
 }());
 
+/**
+ * Whether or not this is a native Android browser.
+ *
+ * @static
+ * @const
+ * @type {Boolean}
+ */
 export const IS_NATIVE_ANDROID = IS_ANDROID && ANDROID_VERSION < 5 && appleWebkitVersion < 537;
 
+/**
+ * Whether or not this is Mozilla Firefox.
+ *
+ * @static
+ * @const
+ * @type {Boolean}
+ */
 export const IS_FIREFOX = (/Firefox/i).test(USER_AGENT);
+
+/**
+ * Whether or not this is Microsoft Edge.
+ *
+ * @static
+ * @const
+ * @type {Boolean}
+ */
 export const IS_EDGE = (/Edge/i).test(USER_AGENT);
+
+/**
+ * Whether or not this is Google Chrome.
```

Comment:
It could be good to mention that this will also include IOS chrome which may have different support then normal.